server.go

```diff
@@ -258,7 +259,7 @@ func (s *Server) recv4(c *ipv4.PacketConn) {
 				ifIndex = cm.IfIndex
 			}
 			if err := s.parsePacket(buf[:n], ifIndex, from); err != nil {
-				log.Printf("[ERR] zeroconf: failed to handle query v4: %v", err)
+				// log.Printf("[ERR] zeroconf: failed to handle query v4: %v", err)
```

Comment:
I think this message should say, because it is very rare and points to a real problem